CMakeLists.txt

```diff
@@ -66,8 +66,6 @@ set (PROJECT_BETA_RELEASE 0)
 
 if (GIT_REVISION)
   set (PROJECT_VERSION_GIT "${GIT_REVISION}")
-else (GIT_REVISION)
-  set (PROJECT_VERSION_GIT "~git")
```

Comment:
@jjnicola did you remove both lines by intention?

Reply:
Yes, it is intentional. In Release versions (if there is no .git folder), the string "~git" must not be added to the lib object filenames.

Reply:
Hmm I suppose this works in  gsa without having to change the cmake file...